debezium-testing/debezium-testing-system/src/main/java/io/debezium/testing/system/tools/kafka/builders/FabricKafkaBuilder.java

```diff
@@ -29,14 +35,20 @@
 import io.strimzi.api.kafka.model.kafka.listener.GenericKafkaListener;
 import io.strimzi.api.kafka.model.kafka.listener.GenericKafkaListenerBuilder;
 import io.strimzi.api.kafka.model.kafka.listener.KafkaListenerType;
+import io.strimzi.api.kafka.model.nodepool.KafkaNodePool;
+import io.strimzi.api.kafka.model.nodepool.KafkaNodePoolBuilder;
+import io.strimzi.api.kafka.model.nodepool.ProcessRoles;
 import io.strimzi.api.kafka.model.zookeeper.ZookeeperClusterSpec;
 import io.strimzi.api.kafka.model.zookeeper.ZookeeperClusterSpecBuilder;
 
 /**
  * This class simplifies building of kafka by providing default configuration for whole kafka or parts of its definition
  */
 public final class FabricKafkaBuilder extends FabricBuilderWrapper<FabricKafkaBuilder, KafkaBuilder, Kafka> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(FabricKafkaBuilder.class);
+    private static Boolean USE_KRAFT;
```

Comment:
Convention in Java is to use upper case names for constant. This is not a constant and is eventually changed later on. Using upper case name can cause the confusion or false assumption in later code edits, so I'd suggest to use lower case letters, or, even better, remove this variable completely.